requirements.txt

```diff
@@ -1,9 +1,13 @@
-pydocstyle==3.*
+# Docker
+docker==4.0.*
```

Comment:
Fix hard docker version please!

Reply:
It will be changed to aiodocker in few commits